code/game/objects/items/weaponry.dm

```diff
@@ -371,7 +371,30 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	to_chat(usr, "<span class='danger'>Message sent. Pray you made the right choice.</span>")
 	usr.log_talk(input, LOG_SAY, tag="Syndicate announcement")
 	deadchat_broadcast(" has messaged the Syndicate using the red phone, \"[input]\" at <span class='name'>[get_area_name(usr, TRUE)]</span>.", "<span class='name'>[usr.real_name]</span>", usr)
-
+	
+/obj/item/phone/banana
+	name = "banana phone"
+	desc = "Emergency telecommunications line to the Clown Planet. Use for emergencies only!"
```

Comment:
Redundant "emergency". Maybe "**Direct** telecommunications line to the Clown Planet. Use for emergencies only!" instead?

Reply:
good idea